### DISCUSSION TITLE
RenderDock

### DISCUSSION THREAD

**Discussion by burtvadr** [skyline-emu/skyline#985] [Q&A] [answered]
I wanted help to understand what renderdock is exactly, and where do I download it, if it's a script or program or anything like that, I'm not a complete programming nerd, but I understand the basics and some fundamentals, I'm here because I love the project and I want to increase my knowledge in the area, if anyone can help me it would be cool, my discord "fcykgvhbik#5698"

**Comment by PixelyIon** [comment, marked as answer]
You should probably just ask things like this on Discord rather than GitHub but you're likely talking about [`RenderDoc`](https://renderdoc.org/), it's a graphics debugger. It's mainly for people who're fairly familiar with the graphics pipeline and not really a tool for everyone. Anyway, I hope that answered your question.
